hetmech/test_degree_weight.py

```diff
@@ -19,14 +18,16 @@ def test_disease_gene_example_dwwc():
 
     # Compute GiGaD path count and DWWC matrices
```

Comment:
because there is a repeated node in the metapath (G) and because this notebook uses `dwwc` and not `dwpc`, the object computed here (`dwwc(graph, metapath, damping=0.5`) is not a path count but a walk count.